Contents/Code/__init__.py

```diff
@@ -105,17 +106,18 @@ def Start():
   HTTP.Headers['Accept-Language'] = 'en-us'
 
 ### Assign unique ID ###
-def Search(results, media, lang, manual, movie):
```

Comment:
GitHub shows this as a change to the method name for Search, but it's not. There's still a Search method, this is just a new one in addition.